src/providers/krb5/krb5_ccache.c

```diff
@@ -54,6 +55,11 @@ errno_t switch_to_user(void)
         return errno;
     }
 
+    ret = setgroups(0, NULL);
```

Comment:
There are several reasons why this is not needed and, most probably, won't work.

First of all, those functions - 'switch_to_*()' - are exclusively used by 'krb5_child' only.
And the first thing 'krb5_child' does - it calls `privileged_krb5_setup()`.
First thing `privileged_krb5_setup()` does - it calls `setgroups(0, NULL)` if appropriate.
So groups are either already dropped, or can't be dropped (see comment https://github.com/SSSD/sssd/blob/c6294f5ff32a917bb1286afcc23f92f4f8ebe010/src/providers/krb5/krb5_child.c#L3997)

Moreover, next thing 'krb5_child' does - `sss_drop_all_caps()`.
Due to this `setgroups()` here will fail with `EPERM`

Reply:
I'm not sure if this would make 'rpmlint' happy, but maybe addition of `else keep-SSSD-group-only` in `privileged_krb5_setup()` would be better.

But imo it's not worth bothering.

Reply:
Hi Alexey, thanks for the detailed explanation.

Adding the `else` part to `privileged_krb5_setup()` will not help because rpmlint just scans the shared library for function calls. In `libsss_krb5_common.so` it finds the the `setresuid()` inside `switch_to_xxx()` without a `setgroups()` call, and `privileged_krb5_setup()` is not part of this library.

I will just add an exemption for this error. What's about the other patch? I will update this PR if it makes sense.

Reply:
> What's about the other patch?

d-bus policy patch looks good to me (but will require a 2nd reviewer).